docker-compose.yml

```diff
@@ -321,15 +322,45 @@ services:
       KAFKA_CLUSTERS_0_JMXPORT: 9997
 
   clickhouse-server:
-    image: clickhouse/clickhouse-server:23.5
+    image: clickhouse/clickhouse-server:24.3
     networks:
       - router_net
     ports:
       - "9000"
       - "8123:8123"
+    volumes:
+      - ./crates/analytics/docs/clickhouse/scripts:/docker-entrypoint-initdb.d
     profiles:
       - analytics
     ulimits:
       nofile:
         soft: 262144
-        hard: 262144
\ No newline at end of file
+        hard: 262144
+  
+  hyperswitch-control-center:
+    image: juspaydotin/hyperswitch-control-center:v1.29.9
+    networks:
+      - router_net
+    ports:
+      - "9000:9000"
+    environment:
+      apiBaseUrl:  http://localhost:8080
+      sdkBaseUrl: http://localhost:9050/HyperLoader.js
+
+  hyperswitch-web-sdk:
+    build:
+      dockerfile_inline: |
+        FROM node:lts
+        RUN git clone https://github.com/juspay/hyperswitch-web.git --depth 1
+        WORKDIR hyperswitch-web
+        RUN npm i --force
+    command: bash -c 'npm run re:build && npx run webpack serve --config webpack.dev.js --host 0.0.0.0'
+    networks:
+      - router_net
+    ports:
+      - "9050:9050"
+    environment:
+      sdkEnv: local
+      envSdkUrl: http://localhost:9050
+      envBackendUrl: http://localhost:8080
+
```

Comment:
```suggestion
      envLoggingUrl: http://localhost:8207
```

Reply:
#4372 Need to pick this up as well tho...